src/routes/blog/posts/+layout.svelte

```diff
@@ -11,7 +12,7 @@
 	$: ({ title, thumbnail, author, date } = data);
 </script>
 
-<Metadata title="Files • {title}" image={thumbnail} />
+<Metadata title={$_("metadata.blog_page", defaultI18nValues)} image={thumbnail} />
```

Comment:
While this is a good change, it's generally better to keep PRs focused on a single area of change rather than multiple things at once (don't worry, i struggle with this myself lol)

Reply:
I renamed the PR.